src/AvatarStack.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {get, COMMON} from './constants'
+import theme from './theme'
+
+const rightStyles = `
+  .body {
+    right: 0;
+    flex-direction: row-reverse;
+
+    &:hover .avatar {
+      margin-right: 0;
+      margin-left: 3px;
+    }
+  }
+
+  .avatar.avatar-more {
```

Comment:
alignRight seems a little off with 4 avatars:
https://codesandbox.io/s/staging-currying-25pvm

It looks like maybe because this is .avatar.avatar-more?
```suggestion
  .avatar-more {
```